src/components/features/FeatureOne.astro

```diff
@@ -1,63 +1,208 @@
-            directly within your website or app. This increases convenience and efficiency
-            for your users, and helps to boost sales for your business
+          <div>
+            <div
+              class="flex items-center justify-center w-12 h-12 text-white shadow-indigo-500/20 bg-gradient-to-tr shadow-2xl from-vulcan-900 to-vulcan-700 rounded-xl"
+            >
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+              <path stroke-linecap="round" stroke-linejoin="round" d="M7.217 10.907a2.25 2.25 0 100 2.186m0-2.186c.18.324.283.696.283 1.093s-.103.77-.283 1.093m0-2.186l9.566-5.314m-9.566 7.5l9.566 5.314m0 0a2.25 2.25 0 103.935 2.186 2.25 2.25 0 00-3.935-2.186zm0-12.814a2.25 2.25 0 103.933-2.185 2.25 2.25 0 00-3.933 2.185z" />
+            </svg>
+            
+            </div>
+            <p class="mt-5 text-lg font-medium text-white lg:text-xl">
+              Build sharing
+            </p>
           </div>
-          <div class="mt-12 r">
-            <img
-              alt="Lexington thumbnail"
-              class="rounded-2xl shadow-2xl shadow-vulcan-900/50 w-full"
-              src="/assets/card2.svg"
-            />
+          <div class="mt-2 text-base text-slate-400">
+            Share builds with your team for active engagement and feedback, promoting a collaborative development cycle.
+          </div>
+        </li>
+        <li>
+          <div>
+            <div
+              class="flex items-center justify-center w-12 h-12 text-white shadow-indigo-500/20 bg-gradient-to-tr shadow-2xl from-vulcan-900 to-vulcan-700 rounded-xl"
+            >
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+              <path stroke-linecap="round" stroke-linejoin="round" d="M7.5 14.25v2.25m3-4.5v4.5m3-6.75v6.75m3-9v9M6 20.25h12A2.25 2.25 0 0020.25 18V6A2.25 2.25 0 0018 3.75H6A2.25 2.25 0 003.75 6v12A2.25 2.25 0 006 20.25z" />
+            </svg>
+            
+            </div>
+            <p class="mt-5 text-lg font-medium text-white lg:text-xl">
+              Build & project insights
+            </p>
+          </div>
+          <div class="mt-2 text-base text-slate-400">
+            We provide teams the data and tools to make informed decisions, detect regressions, and maintain project health.
           </div>
         </li>
       </ul>
     </div>
+    <div
+      class="grid grid-cols-1 mt-12 lg:grid-cols-8 lg:mt-24 items-start gap-12 lg:gap-32"
+    >
+      <div class="col-span-3">
+        <h2 class="mt-6 text-3xl font-normal text-white">Return of Investment</h2>
+        <p class="mt-5 text-lg text-slate-400">
+          Organizations face expenses due to developer inefficiencies. Tuist
+          Cloud optimizes workflows and infrastructure, mitigating frequent
+          inefficiencies like:
```

Comment:
```suggestion
          Cloud optimizes workflows and infrastructure, making developers faster and their work more enjoyable by:
```

Inefficiencies are mentioned twice here. Also, let's mention that developers will enjoy work much more if they don't have to wait for things to build? 🙂